### PR TITLE
Fixed aliases

### PR DESCRIPTION
**Pull request by heftymouse**
ye

### CODE REVIEW

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/App.java [hunk 1]
```diff
@@ -124,6 +124,7 @@ public static void main(String[] args) {
 				.filter(event -> event.getMessage().getContent().startsWith("!eyeyoureadyforit "))
 				.subscribe(event -> {
 					if (event.getMessage().getContent().equals("!eyeyoureadyforit reload")) {
+						LOGGER.info("Reloading data...");
```
Comment (BoogieMonster1O1) [inline app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/App.java:127]: CONCERN NO SEND MESSAGE IN DISCORD AS WELL OR BAD

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java [hunk 2]
```diff
@@ -2,21 +2,10 @@
...
 import java.nio.file.Path;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Random;
+import java.sql.*;
```
Comment (BoogieMonster1O1) [inline app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java:7]: ir its fine now but disable this

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java [hunk 3]
```diff
@@ -2,21 +2,10 @@
...
+import java.util.*;
```
Comment (BoogieMonster1O1) [inline app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java:8]: ditto

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java [hunk 4]
```diff
@@ -85,17 +74,8 @@ public static void reload() {
...
-						int l;
-						try {
-							l = Array.getLength(aliases);
-						} catch (RuntimeException e) {
-							l = 0;
-						}
-						for (int i = 0; i < l; i++) {
-							aliases.add(Array.get(aliasesObj, i).toString());
-						}
+						Object[] aliasesSqlArray = (Object[]) set.getArray("ALIASES").getArray();
```
Comment (BoogieMonster1O1) [inline app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java:77]: pls inline this irrit local variable

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/App.java (hunk 1 above)
Comment (heftymouse) [reply app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/App.java:127]: dun

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java (hunk 2 above)
Comment (heftymouse) [reply app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java]: dun

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java (hunk 3 above)
Comment (heftymouse) [reply app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java]: dun

File: app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java (hunk 4 above)
Comment (heftymouse) [reply app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/data/EyeEntry.java]: dun

**Review by BoogieMonster1O1 — APPROVED** [review]

**Pull request merged**